src/net.h

```diff
@@ -982,7 +982,7 @@ class CConnman
      */
     bool AlreadyConnectedToAddress(const CAddress& addr);
 
-    bool AttemptToEvictConnection();
+    bool AttemptToEvictConnection(bool force);
```

Comment:
I think it would be nice to document the effect of `force` both here and for `SelectNodeToEvict`